installation.rst

```diff
@@ -204,9 +204,9 @@ basic Slurm cluster:
   cloud environment. Currently available ones are AWS and LXD.
 - ``slurm-core/series/``: overlays to define the OS of the Slurm components,
   the possible options are CentOS7 and Ubuntu Focal (20.04).
-- ``slurm-core/charms/``: overlays to change the source of the charm. By
-  default, the bundle gets the ``latest/stable`` channel from Charmhub. The
-  overlays in this directory allows ut to change the source to either
+- ``slurm-core/charms/``: overlays to change the source of the charms. By
+  default, the bundle gets from the ``latest/stable`` channel from Charmhub.
```

Comment:
This is a little unclear: "By default, the bundle gets from the ``latest/stable`` channel from Charmhub."

I'm not quite sure _what_ it's getting from that channel.

Reply:
Is it clear now?